src/modules/modals/Modals.vue

```diff
@@ -109,26 +113,30 @@ export default {
 		subscribe('tables:view:delete', view => { this.viewToDelete = view })
 
 		// columns
-		subscribe('tables:column:create', () => { this.showCreateColumn = true })
-		subscribe('tables:column:edit', column => { this.columnToEdit = column })
-		subscribe('tables:column:delete', column => { this.columnToDelete = column })
+		subscribe('tables:column:create', columnInfo => { this.createColumnInfo = columnInfo })
+		subscribe('tables:column:edit', columnInfo => { this.columnToEdit = columnInfo })
+		subscribe('tables:column:delete', columnInfo => { this.columnToDelete = columnInfo })
 
 		// rows
-		subscribe('tables:row:create', columns => { this.columnsForRow = columns })
-		subscribe('tables:row:edit', row => { this.editRow = row })
-		subscribe('tables:row:delete', rows => { this.rowsToDelete = rows })
+		subscribe('tables:row:create', columnsInfo => { this.columnsForRow = columnsInfo })
```

Comment:
With the adjustment where this is emitted it should probably be rather this:
```suggestion
		subscribe('tables:row:create', ({ columns }) => { this.columnsForRow = columns })
```

Reply:
Similar adjustments we'd need for the other ones

Reply:
I don't think this works, since we want the entire object, not just the `columns` property

Reply:
Sorry I somehow missed the updated usage of it, nevermind about the earlier comment